packages/analytics-js/src/services/StoreManager/StoreManager.ts

```diff
@@ -65,11 +65,10 @@ class StoreManager implements IStoreManager {
         secure: loadOptions.secureCookie,
         domain: loadOptions.setCookieDomain,
         sameDomainCookiesOnly: loadOptions.sameDomainCookiesOnly,
-        enabled: true,
       },
-      localStorageOptions: { enabled: true },
-      inMemoryStorageOptions: { enabled: true },
-      sessionStorageOptions: { enabled: true },
+      localStorageOptions: {},
+      inMemoryStorageOptions: {},
+      sessionStorageOptions: {},
     };
```

Comment:
These are already present in the default options of the respective module. It doesn't make sense to pass any values unless we're overriding them.